src/hooks/use-timeout.ts

```diff
@@ -0,0 +1,22 @@
+// h/t Dan Abramov - https://overreacted.io/making-setinterval-declarative-with-react-hooks/
+import { useEffect, useRef } from 'react'
+
+export function useInterval(callback: () => void, delay?: number | null): void {
```

Comment:
Is the file intentionally named something different than the function?

Reply:
No, the function was renamed without the file being renamed. Thanks for pointing it out.